src/main/java/codeu/model/data/BotStructure.java

```diff
@@ -0,0 +1,49 @@
+package codeu.model.data;
+
+/** Class representing the BotStructure, which establishes the User that represents the bot. */
```

Comment:
I'm not sure I understand what this is going to be used for, and I don't see any other code in your PR that references it.

Can you explain how this fits into the whole picture?